src/pages/shift-left-the-good-parts.tsx

```diff
@@ -0,0 +1,341 @@
+                }
+                rightColumn={
+                    <Video
+                        host="self"
+                        source={{
+                            mp4: 'blog/shift-left/10-code-reviews',
+                            webm: 'blog/shift-left/10-code-reviews',
+                        }}
+                        title="Find references in code reviews"
+                        caption="Go-to-definition and find references at work."
+                        loop={true}
+                    />
+                }
+            />
+        </ContentSection>
+
+        <div className="bg-gradient-venus">
+            <ContentSection className="py-7">
+                <section className="max-w-800 mx-auto text-md-center">
+                    <h2>Ready to shift left?</h2>
+
+                    <p className="my-5 max-w-650 mx-md-auto">
+                        Shifting left means getting a head start on some of the most important aspects of the
+                        development process. Both the general philosophy and the specific process changes of shift left
+                        can be boosted by using tools that enable the developer to take full charge of shipping a new
+                        feature or bug fix.
+                    </p>
+
+                    <div className="d-flex flex-column align-items-center">
+                        <Link href="/demo" passHref={true}>
+                            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                            <a
+                                title="Request a demo"
+                                className="btn btn-primary col-md-3"
+                                data-button-style={buttonStyle.text}
+                                data-button-location={buttonLocation.body}
+                                data-button-type="cta"
+                            >
+                                Request a demo
+                            </a>
+                        </Link>
+                        <Link href="/get-started/self-hosted" passHref={true}>
+                            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                            <a
+                                title="Try Sourcegraph now"
+                                className="mt-4 font-weight-bold text-center col-md-4"
+                                data-button-style={buttonStyle.text}
+                                data-button-location={buttonLocation.body}
+                                data-button-type="cta"
+                            >
+                                Try Sourcegraph now
+                                <ArrowRightIcon className="ml-2" />
+                            </a>
+                        </Link>
+                    </div>
+                </section>
+            </ContentSection>
+        </div>
+
+        <ContentSection color="white" className="container max-w-900 py-7 px-0">
```

Comment:
The Figma doesn't show this having a smaller width constraint so I don't think we should add the max width. Was this a request?
```suggestion
        <ContentSection color="white" className="container py-7 px-0">
```